backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -18,11 +18,12 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/kubeflow/pipelines/backend/src/apiserver/template"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/kubeflow/pipelines/backend/src/apiserver/template"
```

Comment:
nit: revert formatting changes unrelated to this PR?

We should probably format all the code again and enforce it in presubmit tests, but for this PR, let's not include them, so it's easier to review. (However, to save your efforts, for any file that you modified, I'm OK with keeping formatting changes.)

Reply:
I will clean it up,  no worries.